src/main/webapp/script.js

```diff
@@ -406,8 +406,8 @@ function createAnswerForm(questionID, repName, postFromFeed, tabName) {
     var formDiv = document.getElementById("answerForm" + questionID);
 
     var ansForm = document.createElement("form");
-    ansForm.setAttribute("action", `/rep_answer?postId=${questionID}&repName=${encodeURI(repName)}
-        &feed=${postFromFeed}&tabName=${tabName}`);
+    ansForm.setAttribute("action", 
+        `/rep_answer?postId=${questionID}&repName=${encodeURI(repName.trim())}&feed=${postFromFeed}&tabName=${tabName}`);
```

Comment:
The reason for this is because the url was getting messed up because when you break the line, it adds a ton of spaces